src/config/schema.ts

```diff
@@ -86,7 +80,7 @@ export type Chain = z.infer<typeof chainSchema>;
 
 // Ensure that the contracts are loaded from the package @api3/contracts if not specified.
 export const chainsSchema = z
-  .record(optionalChainSchema)
+  .record(z.string(), optionalChainSchema)
```

Comment:
Interesting seems like they have backtracked their decision on deprecating `superRefine()`. I had to convert all of them to `check()` when I was migrating to v4 🤷‍♂️

Reply:
> they have backtracked their decision on deprecating superRefine()

Yes, it looks like. However, it’s probably better to replace `.addIssue` with `.issues.push`, as `.addIssue` is marked as deprecated: https://zod.dev/v4/changelog?id=deprecates-addissue-and-addissues.